paddle/framework/framework.proto

```diff
@@ -143,3 +143,13 @@ message BlockDesc {
 // https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/program.md
 // for more details.
 message ProgramDesc { repeated BlockDesc blocks = 1; }
+
+message OpPlacement {
+  optional string name = 1;
+  optional string device = 2;
+}
+
+message ExecutionPlan {
+  optional ProgramDesc program = 1;
+  repeated OpPlacement op_placement = 2;
+}
```

Comment:
So, how to find the correspondence between OpPlacement in ExecutionPlan and OpDesc in ProgramDesc?
Are the number and order of operators in ExecutionPlace and ProgramDesc the same?

Reply:
The number will be the same, each OP will have one placement. The order does not have to be the same, otherwise the "name" field in `OpPlacement` is not necessary.

Reply:
Program{Block{Op}}. A Program has many blocks. A block has many ops.

However, the Program has many operator placements.   We cannot get a one-to-one map by this data structure.

Reply:
@reyoung 

> However, the Program has many operator placements. We cannot get a one-to-one map by this data structure.

Sorry I don't fully get this point, I thought different OPs have different names?